src/components/DecisioningEngine/index.js

```diff
@@ -48,6 +48,26 @@ const createDecisioningEngine = ({ config, createNamespacedStorage }) => {
       },
       onComponentsRegistered(tools) {
         applyResponse = createApplyResponse(tools.lifecycle);
+        if (personalizationStorageEnabled) {
+          const storage = createNamespacedStorage(
+            `${sanitizeOrgIdForCookieName(orgId)}.decisioning.`
+          );
+          consent
```

Comment:
I think it's fine to have `.awaitConsent()` here in the `index.js` file.  But I wonder if it makes sense to move the listener within `eventRegistry` instead? 🤔 There are pros/cons.